docs/maintainers/concept-exercise-style-guide.md

```diff
@@ -0,0 +1,56 @@
+# Formatting and Style Guide for Concept Exercise Files
```

Comment:
Could you add a link to this new document to the [_sidebar.md document](https://github.com/exercism/v3/blob/master/docs/maintainers/_sidebar.md)?